internal/gatewayapi/testdata/gateway-with-listener-with-valid-multiple-tls-configuration.out.yaml

```diff
@@ -28,20 +28,16 @@ gateways:
     - attachedRoutes: 1
       conditions:
       - lastTransitionTime: null
-        message: Sending translated listener configuration to the data plane
-        reason: Programmed
-        status: "True"
-        type: Programmed
-      - lastTransitionTime: null
-        message: Listener has been successfully translated
-        reason: Accepted
-        status: "True"
-        type: Accepted
-      - lastTransitionTime: null
-        message: Listener references have been resolved
-        reason: ResolvedRefs
-        status: "True"
+        message: 'Secret envoy-gateway/tls-secret-ecdsa-1 must contain valid tls.crt
```

Comment:
we should update the cert material here then, because the testdata is for a valid case